core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadWriteLDAPUserStoreConstants.java

```diff
@@ -68,47 +70,58 @@
                 .readLDAPGroupsDescription);
-        setProperty(UserStoreConfigConstants.writeGroups, "Write Groups", "true", UserStoreConfigConstants.writeGroupsDescription);
+        setProperty(UserStoreConfigConstants.writeGroups, "Write Groups", "true",
+                UserStoreConfigConstants.writeGroupsDescription);
         setProperty(UserStoreConfigConstants.groupSearchBase, "Group Search Base", "ou=Groups,dc=wso2,dc=org",
                 UserStoreConfigConstants.groupSearchBaseDescription);
         setProperty(UserStoreConfigConstants.groupEntryObjectClass, "Group Entry Object Class", "groupOfNames",
                 UserStoreConfigConstants.groupEntryObjectClassDescription);
         setProperty(UserStoreConfigConstants.groupNameAttribute, "Group Name Attribute", "cn",
                 UserStoreConfigConstants.groupNameAttributeDescription);
-        setProperty(UserStoreConfigConstants.groupNameSearchFilter, "Group Search Filter", "(&amp;(objectClass=groupOfNames)(cn=?))",
+        setProperty(UserStoreConfigConstants.groupNameSearchFilter, "Group Search Filter",
+                "(&amp;(objectClass=groupOfNames)(cn=?))",
                 UserStoreConfigConstants.groupNameSearchFilterDescription);
         setProperty(UserStoreConfigConstants.groupNameListFilter, "Group List Filter", "(objectClass=groupOfNames)",
                 UserStoreConfigConstants.groupNameListFilterDescription);
         setProperty(roleDNPattern, "Role DN Pattern", "", roleDNPatternDescription);
 
-        setProperty(UserStoreConfigConstants.membershipAttribute, "Membership Attribute", "member", UserStoreConfigConstants.membershipAttributeDescription);
-        setProperty(UserStoreConfigConstants.memberOfAttribute, "Member Of Attribute", "", UserStoreConfigConstants.memberOfAttribute);
-        setProperty("BackLinksEnabled", "Enable Back Links", "false", " Whether to allow attributes to be result from" +
-                "references to the object from other objects");
-
-        setProperty(UserStoreConfigConstants.usernameJavaRegEx, "Username RegEx (Java)", "[a-zA-Z0-9._-|//]{3,30}$", UserStoreConfigConstants.usernameJavaRegExDescription);
-        setProperty(UserStoreConfigConstants.usernameJavaScriptRegEx, "Username RegEx (Javascript)", "^[\\S]{3,30}$", UserStoreConfigConstants.usernameJavaRegExDescription);
+        setProperty(UserStoreConfigConstants.membershipAttribute, "Membership Attribute", "member",
+                UserStoreConfigConstants.membershipAttributeDescription);
+        setProperty(UserStoreConfigConstants.memberOfAttribute, "Member Of Attribute", "",
+                UserStoreConfigConstants.memberOfAttribute);
+        setProperty("BackLinksEnabled", "Enable Back Links", "false",
+                "Whether to allow attributes to be result from references to the object from other objects");
+
+        setProperty(UserStoreConfigConstants.usernameJavaRegEx, "Username RegEx (Java)", "[a-zA-Z0-9._-|//]{3,30}$",
```

Comment:
Can you check the line length whether those are exceeding the maximum line length?

Reply:
Yes. It exceeded the maximum line length. That is why it was formatted.